src/uct/ib/rc/base/rc_iface.c

```diff
@@ -935,50 +935,51 @@ ucs_status_t uct_rc_iface_qp_connect(uct_rc_iface_t *iface, struct ibv_qp *qp,
     return UCS_OK;
 }
 
-ucs_status_t uct_rc_iface_common_event_arm(uct_iface_h tl_iface,
-                                           unsigned events, int force_rx_all)
+int uct_rc_iface_should_poll_rx(uct_rc_iface_t *iface, unsigned events,
```

Comment:
uct_rc_iface_should_arm_rx_cq
since we use it to decide if need to arm the cq or not